pom.xml

```diff
@@ -52,7 +52,7 @@
 		<appuiVersion>1.13.2-SNAPSHOT</appuiVersion>
 		<htmlformentryVersion>4.0.1-SNAPSHOT</htmlformentryVersion>
 		<htmlformentryuiVersion>2.0.0-SNAPSHOT</htmlformentryuiVersion>
-		<coreappsVersion>1.31.2-SNAPSHOT</coreappsVersion>
+		<coreappsVersion>1.32.0-SNAPSHOT</coreappsVersion>
```

Comment:
Did you intend to  update  coreappversion , this is already released , so you can revert this and fix travis failure thanks